Client/Application/AppDelegate.swift

```diff
@@ -146,6 +149,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             await FeatureManagement.fetchConfiguration()
         }
         MMP.sendSession()
+        searchesCounter.subscribe(self) { searchCount in
```

Comment:
@ecosia/ios @d4r1091 No need to review yet, but wanted your opinion here, I need to globally subscribe somewhere to the counter so that search events are handled. Not sure I like adding it to AppDelegate but see no better candidates given this happens across the whole app and not in any specific class. What do you think?

Reply:
Hey @lucaschifino 👋 , I would lean towards your choice simply because of the `MMP` reference throughout the app is so far only present in the `Welcome` context and the `NTPImpact`.
For reference though, another long-lived object (also instantiated by the `AppDelegate`) is the `BrowserViewController` in case we want another indication.
However, considering the upgrade, this place is the best to facilitate the conflicts as well.